server/intelligence-service/src/config.py

```diff
@@ -1,5 +1,17 @@
+from dotenv import load_dotenv
+import os
+
+load_dotenv()
+
 class Settings:
     APP_NAME: str = "Intelligence Service"
+    # Non-Azure OpenAI Configuration
+    OPENAI_API_KEY = os.getenv("OPENAI_API_KEY")
+
+    # Azure OpenAI Configuration
+    AZURE_OPENAI_API_KEY = os.getenv("AZURE_OPENAI_API_KEY")
+    AZURE_OPENAI_ENDPOINT = os.getenv("AZURE_OPENAI_ENDPOINT")
+    AZURE_OPENAI_API_VERSION = os.getenv("AZURE_OPENAI_API_VERSION")
```

Comment:
I don't think that we need those here, LangChain should take them from the environment automatically